src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Craftable Deepslate Ore/pack.mcmeta

```diff
@@ -0,0 +1,12 @@
+{
+    "pack": {
+        "pack_format": 7,
+        "description": "RedPack by BisUmTo, Gnottero & DukeEdivad05 - Ore Upgrade v0.1.0"
+    },
+    "redpack": {
+        "id": "128358",
+        "video": "KVIxcL8rNC8",
+        "image": "https://img.youtube.com/vi/0GiWWcVG6x8/maxresdefault.jpg",
+        "group": "more_craftings"
```

Comment:
Non sono certo se tenerlo nel gruppo more_craftings, ma per ora va più che bene lì in quanto il sito web di RedPack non è ancora pubblico.